releasenotes/notes/fix-phase-in-remove-id-equivalent-6480da0c62f20df1.yaml

```
@@ -0,0 +1,8 @@
+---
+fixes:
+  - |
+    Fixed a bug in :class:`.RemoveIdentityEquivalent` transpiler pass, where the
+    unitary gates close to identity up to a global phase were removed from the circuit,
+    but the global phase of the circuit was not updated. In particular,
+    :class:`.RemoveIdentityEquivalent` now removes non-parameterized :class:`.GlobalPhaseGate`
+    gates.
```

To me the adjective "unitary" here could mean we refer to `UnitaryGate` objects, maybe removing it is better (since it could have been any other gate)?
```suggestion
    Fixed a bug in the :class:`.RemoveIdentityEquivalent` transpiler pass, where
    gates close to identity up to a global phase were removed from the circuit,
    but the global phase of the circuit was not updated. In particular,
    :class:`.RemoveIdentityEquivalent` now removes non-parameterized :class:`.GlobalPhaseGate`
    gates.
```

---

Sure, done in [68e6a7f](https://github.com/Qiskit/qiskit/pull/13785/commits/68e6a7fca205e994ea7e1b0d95a8c082a9644235).